c2corg_ui/templates/outing/edit.html

```diff
@@ -397,6 +397,7 @@ <h2 class="heading show-phone">
               <label translate>public_transport</label>
               <input type="checkbox" ng-model="outing.public_transport" ng-checked="outing.public_transport == true">
               <span ng-click="editCtrl.pushToArray(outing, 'public_transport', !outing.public_transport, $event)"></span>
+              <a href="http://changerdapproche.org/le-concours/presentation.html" target="_blank" ng-if="outing.public_transport == true" translate>Participate to MW contest</a>
```

Comment:
How stupid :confounded: 
Fast copy and paste (have a look at line 398... :wink:)